### DISCUSSION TITLE
Adding new language to the Language OptionButton

### DISCUSSION THREAD

**Discussion by Astrono2** [mbrlabs/Lorien#14] [Q&A] [answered]
I'm trying to make a spanish localization, but I can't find where the other items are added to the OptionButton. Might be a dumb question from not having much experience with Godot's UI.

**Comment by mbrlabs** [comment, marked as answer]
Cool, yeah it's a bit hidden. It's in SettingsDialog.tscn. Click on that OptionButton:

![i1](https://user-images.githubusercontent.com/8750135/120811277-170f5800-c54c-11eb-999d-7087c66444c3.PNG)

Then in the toolbar you will see a button called `Items`. Click and add the language there:
![i2](https://user-images.githubusercontent.com/8750135/120811366-34442680-c54c-11eb-9e7f-121df6bdd203.PNG)


In the future i'm going to make this automatic probably so that you only have to add the localization file and it will get listed in the UI automatically.

**Reply by Astrono2** [reply]
Ohhh I see, thank you!
